src/main/java/frc/robot/RobotContainer.java

```diff
@@ -191,7 +191,10 @@ private void driveBindings() {
             mechAButton.onTrue(new DropperChooserCommand(swerveSubsystem, rollerSubsystem, tiltedElevatorSubsystem));
 
             driveController.getFieldResetButton().onTrue(new InstantCommand(swerveSubsystem::resetFieldAngle, swerveSubsystem));
-            driveController.getChargingStationLockButton().onTrue(new InstantCommand(swerveSubsystem::toggleChargingStationLocked, swerveSubsystem));
+            driveController.getChargingStationLockButton()
+                .onTrue(new InstantCommand(swerveSubsystem::toggleChargingStationLocked, swerveSubsystem));
+            driveController.getSwerveRelativeFeedbackButton()
```

Comment:
i don't think this should go on the drive controller, i think it should go on the switchboard